models/engine/db_storage.py

```diff
@@ -74,3 +74,17 @@ def reload(self):
     def close(self):
         """call remove() method on the private session attribute"""
         self.__session.remove()
+
+    def get(self, cls, id):
+        """function get() that return the object based on an ID"""
+        if cls and id:
+            value = "{}.{}".format(cls, id)
+            obj_list = self.all(cls)
+            for instance in obj_list.values():
+                if instance.id in value:
+                    return instance
+        return (None)
+
+    def count(self, cls=None):
+        """fonction count() that return the lenght of all objets"""
+        return (len(self.all(cls)))
```

Comment:
What if cls doesn't exist, like, you get arg like "Beach" and can't be treated by all() method ?

Reply:
Can you see it now ?

Reply:
yes i can see it

Reply:
If cls doesn't exist it's will be none or it will not be treated

Reply:
Ok nice!